src/openforms/authentication/tests/test_views.py

```diff
@@ -140,6 +148,44 @@ def test_plugin_start_failure_redirects(self):
         expected.args[BACKEND_OUTAGE_RESPONSE_PARAMETER] = "plugin1"
         self.assertEqual(furl(response["Location"]), expected)
 
+    @override_settings(CORS_ALLOW_ALL_ORIGINS=True)
+    def test_plugin_start_failure_not_enabled(self):
+        register = Registry()
+        register("plugin1")(Plugin)
+        plugin = register["plugin1"]
+
+        config = GlobalConfiguration.get_solo()
+        config.plugin_configuration = {
+            "authentication": {"plugin1": {"enabled": False}}
+        }
+        config.save()
```

Comment:
we've sort of shifted patterns - rather than updating the database (and cache!) values with django-solo models, we mock the `GlobalConfiguration.get_solo` call and return an unsaved instance of `GlobalConfiguration` with the relevant configuration.

This ensures that tests stay properly isolated - with cache enabled, you have to be really careful to either override the cache settings for tests or reset the GlobalConfiguration, and it can even bleed into other test-cases. The mock-route is the most future proof approach in tests for this.